projects/plugins/jetpack/modules/publicize.php

```diff
@@ -17,34 +17,19 @@
 
 // phpcs:disable Universal.Files.SeparateFunctionsFromOO.Mixed -- TODO: Move classes to appropriately-named class files.
 
+use Automattic\Jetpack\Status\Host;
+
 /**
  * Class Jetpack_Publicize
  */
 class Jetpack_Publicize {
-	/**
-	 * Current status about Jetpack modules.
-	 *
-	 * @var Automattic\Jetpack\Modules
-	 */
-	public $modules;
```

Comment:
I noticed that the `$this->modules` field is no longer used, so I'm removing it. Existing usages were gradually removed (in PRs like #35781) and eventually usage dropped to zero, unnoticed.